cmd/k8s-operator/operator.go

```diff
@@ -556,6 +581,23 @@ func managedResourceHandlerForType(typ string) handler.MapFunc {
 	}
 }
 
+// isManagedByTSRecorder implements [handler.MapFunc]
+func isManagedByTSRecorder(_ context.Context, o client.Object) []reconcile.Request {
+	for _, owner := range o.GetOwnerReferences() {
+		if owner.Kind == "TSRecorder" {
+			return []reconcile.Request{
+				{
+					NamespacedName: types.NamespacedName{
+						Name: owner.Name,
+					},
+				},
+			}
+		}
+	}
+
+	return nil
+}
+
```

Comment:
You might be able to achieve the same with https://github.com/kubernetes-sigs/controller-runtime/blob/e6c3d139d2b6c286b1dbba6b6a95919159cfe655/pkg/handler/enqueue_owner.go#L50